internal/bminventory/inventory.go

```diff
@@ -5247,3 +5250,15 @@ func (b *bareMetalInventory) ListClusterHosts(ctx context.Context, params instal
 	}
 	return installer.NewListClusterHostsOK().WithPayload(hostList)
 }
+
+func (b *bareMetalInventory) GetReadyAndApprovedHostsSummary(clusterID strfmt.UUID) (registered, approved int, err error) {
```

Comment:
not sure i understand difference from the function name between 
 GetReadyAndApprovedHostsSummary and GetReadyAndApprovedHosts , can we have some explanation somewhere? how to choose the right one?

Reply:
GetReadyAndApprovedHostsSummary returns counts, and GetReadyAndApprovedHosts returns the hosts.
GetReadyAndApprovedHostsSummary is more efficient since it offloads the work to the DB server.
Use GetReadyAndApprovedHosts if the hosts are actually needed